core/src/Kokkos_CudaSpace.hpp

```diff
@@ -57,6 +57,23 @@
 
 #include <Cuda/Kokkos_Cuda_abort.hpp>
 
+#ifdef KOKKOS_IMPL_DEBUG_CUDA_PIN_UM_TO_HOST
+#ifndef KOKKOS_IMPL_DEFINED_CUDA_PIN_UM_TO_HOST_V
+extern bool kokkos_impl_cuda_pin_um_to_host_v;
```

Comment:
@crtrott will this possibly pose any issues with rdc builds? I admit I don't fully understand the nuances of rdc, but my understanding is global variables and `extern` usage has been a generic red flag for a pause and thought I should check.

Reply:
it is a red flag for extern / global variables on the device. These are host only variable.s